inc/Classes/MasterForm.php

```diff
@@ -1205,4 +1209,22 @@ private function get_fieldlength($string)
 
         return 0;
     }
+
+    private function is_field_enum($field_type)
+    {
+        return $this->str_starts_with($field_type, 'enum');
```

Comment:
```suggestion
        return str_starts_with($field_type, 'enum');
```

We now depend on PHP 8, where [str_starts_with](https://www.php.net/manual/de/function.str-starts-with.php) is available.